docs/getting-started/supported-languages-and-tools.md

```diff
@@ -393,7 +393,7 @@ The table below lists all languages and frameworks that Codacy supports and the
 
 ## Codacy wrappers for supported tools
 
-Codacy adds support for new languages and tools by using ["wrapper" Docker images for running each tool](https://github.com/codacy/codacy-example-tool).
+Codacy adds support for new languages and tools by using [a Docker image to wrap each tool](https://github.com/codacy/codacy-example-tool).
```

Comment:
@nicklem Please validate if my suggestion makes sense. 🙏  Feel free to revert it. 😃

Reply:
Thanks @claudiacarpinteiro! I'll tweak this slightly, wee my comment below.